.github/ISSUE_TEMPLATE/collaboration-cycle-request.yml

```diff
@@ -95,24 +104,14 @@ body:
     validations:
       required: true
   - type: dropdown
-    id: visible-changes
+    id: iteration
     attributes:
-      label: Will your work result in visible changes to the user experience?
+      label: Is this an iteration on an existing product/feature?
```

Comment:
I think this question might be confusing. Maybe we could pivot this question to be more like the new VA.gov/VA Mobile App question, where it's either one or the other. Is the product an iteration on an existing product/feature or is a new product/feature?